src/pybreaker.py

```diff
@@ -52,7 +52,7 @@ class CircuitBreaker(object):
     """
 
     def __init__(self, fail_max=5, reset_timeout=60, exclude=None,
-                 listeners=None, state_storage=None, name=None):
+                 listeners=None, state_storage=None, name=None, settings=None):
```

Comment:
Why did you opted for a new `settings` dict instead of just adding a new kwarg for the new setting?

Reply:
in the event there's more settings in the future the kwargs could get messy too

Reply:
> in the event there's more settings in the future the kwargs could get messy too

I think that ship's sailed 😅 

> I can certainly make it a kwarg if you think that's better, let me know

May be a matter of personal taste anyway, but I think adding a new kwarg for this setting might better adhere to the code style currently in use.

Thanks for your patience.

Reply:
No problem, will update :)